tests/math_translator_spec.js

```diff
@@ -229,5 +270,14 @@ describe('MathTranslator (normalizeTranslatedMath)', function() {
         const outputStr = normalizeTranslatedMath(translatedStr, 'en');
         assert.equal(outputStr, normalizedStr);
     });
+
+    it('use \\operatorname{} even for commands supported by KaTeX', function() {
+        const translatedStr = '\\tg \\arctg \\cotg \\ctg';
```

Comment:
Note that these would currently throw a linter error, but I guess it does not hurt to have this here.

Reply:
Thanks for the note. I suggest you update the comment for the rule to mention that this will raise a translations lint error.

Reply:
Will do, thanks!